object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -1,46 +1,61 @@
 = Define Object Stores Examples - Mule 4
 :page-aliases: connectors::object-store/object-store-to-define-a-new-os.adoc
 
-You can define an Object Store globally in the app so it can be referenced by name and shared between multiple components, or you can create an Object Store that only for a particular component.
+You can define an object store globally in the app so that it can be referenced by name and shared between multiple components, or you can create an object store for one particular component.
 
-All Object Store operations can reference a global Object Store. For example, suppose you want to have an audit flow that verifies whether certain users are currently authorized. You might do something like this to reference the Object Store that you have defined:
+All object store operations can reference a global object store. For example, for an audit flow that verifies whether certain users are currently authorized, you can reference the object store that you defined in the app:
 
 `<os:contains id="#[userId]" objectStore="tokensStore" />`
 
 [[global_os]]
-== Define a Global Object Store
+== Defining a Global Object Store
 
-You should use a global Object Store if any of these conditions is true:
+Using a global object store is ideal for:
 
-* You need to share state across components: Because the Object Store can be referenced by name, more than one component can use it. Suppose you have a two instances of the Salesforce connector that are configured in slightly different ways. You can use both instances with the same tokens, avoiding the need to authenticate again.
-* You need to share state across Cluster Nodes: When you are using Mule in cluster mode, you might want to have information that is available to every node in the cluster. Global Object Stores are ideal for that.
-* You need to use information in your app's logic: The Object Store connector is not only capable of defining that store, it can also manipulate it.
+* Sharing state across components
++
+Because the object store is referenced by name, more than one component can use it. Suppose you have a two instances of Salesforce Connector that are configured slightly differently. You can use the same token for both Salesforce Connector instances to avoid the need to reauthenticate.
```

Comment:
```suggestion
Because the object store is referenced by name, more than one component can use it. Suppose you have two instances of Salesforce Connector that are configured slightly differently. You can use the same token for both Salesforce Connector instances to avoid the need to reauthenticate.
```